tests/test_density_ComplexCF.py

```diff
@@ -212,34 +212,34 @@ def test_query_nn(self):
 
         cf = freud.density.ComplexCF(rmax, dr)
         cf.compute(box, ref_points, ref_values, points, values,
-                   query_args={'mode': 'nearest', 'nn': 1})
+                   query_args={'mode': 'nearest', 'num_neighbors': 1})
```

Comment:
Some tests should use mode inference, by removing `'mode': 'nearest'`.

Reply:
Maybe those should be tested in the locality tests instead of here.

Reply:
Yes, I definitely in locality and not here.